rdr_service/resource/tasks.py

```diff
@@ -25,26 +25,25 @@ def batch_rebuild_participants_task(payload, project_id=None):
     :param project_id: String identifier for the GAE project
     :param payload: Dict object with list of participants to work on.
     """
-    ps_bqgen = BQParticipantSummaryGenerator()
-    # Temporarily comment out to improve performance when rebuilding participants.
-    # res_gen = ParticipantSummaryGenerator()
+    res_gen = ParticipantSummaryGenerator()
+    pdr_gen = PDRParticipantSummaryGenerator()
 
+    ps_bqgen = BQParticipantSummaryGenerator()
     pdr_bqgen = BQPDRParticipantSummaryGenerator()
     mod_bqgen = BQPDRQuestionnaireResponseGenerator()
     count = 0
 
     batch = payload['batch']
 
     logging.info(f'Start time: {datetime.utcnow()}, batch size: {len(batch)}')
-    # logging.info(json.dumps(batch, indent=2))
+    logging.info(json.dumps(batch, indent=2))
```

Comment:
Is this meant to remain uncommented, for now?

Reply:
I'll comment it back out.